harness/buildString.js

```diff
@@ -14,3 +14,19 @@ function buildString({ loneCodePoints, ranges }) {
   }
   return result;
 }
+
+function test(regex, string, expression) {
```

Comment:
`test` is a generic name and might bring conflicts in the future.

`testPropertyEscapes` should work. It's ugly but that's the spirit for test262. The beautiful smells.


+1 on `regExpUtils.js`